refactor (#335)
KxSystems/docs: docs/interfaces/q-server-for-odbc3.md

`test.q` provides additional examples of SQL usage, including the create/insert/update/delete statement syntax.

## Custom authentication

> Custom authentication is supported whereby the username and password as set in a DSN or connection string are transformed by a user supplied function. 

Reviewer: ```suggestion
Custom authentication is supported, allowing the username and password specified in a DSN or connection string to be transformed by a user-defined function. 
```
